sites/public/src/components/shared/MapControl.module.scss

```diff
@@ -0,0 +1,32 @@
+$control-border-radius: 100px;
```

Comment:
can you autoformat this file? ty

Can we also localize these variables into .map-control (i.e. put between lines 5 and 6)